Dockerfile.tmpl

```diff
@@ -56,6 +56,14 @@ FROM scratch AS shellcheck
 FROM scratch AS jq
 	COPY --from=ghcr.io/jqlang/jq:1.7.1 /jq /build/bin/
 
+FROM go AS spectral
+	ARG TARGET_GOOS
+	ARG TARGET_GOARCH
+
+	COPY build/spectral /tmp/build
+
+	RUN "/tmp/build" "{{ .data.spectral }}" "${TARGET_GOOS}" "${TARGET_GOARCH}"
+
```

Comment:
This is the template. Am I missing something? Why do you need to add this by hand?